### PR TITLE
drivers: eeprom: mchp: Bug fix

### PR DESCRIPTION
**Pull request by Manimaran-A**
Updated the DT macro to read EEPROM size property.

### CODE REVIEW

File: drivers/eeprom/eeprom_mchp_xec.c [hunk 1]
```diff
@@ -325,7 +325,7 @@ PINCTRL_DT_INST_DEFINE(0);
 
 static const struct eeprom_xec_config eeprom_config = {
 	.regs = (struct eeprom_xec_regs * const)DT_INST_REG_ADDR(0),
-	.size = DT_INST_REG_SIZE(0),
+	.size = DT_PROP(DT_DRV_INST(0), size),
```
Comment (henrikbrixandersen) [inline drivers/eeprom/eeprom_mchp_xec.c:328]:
```suggestion
	.size = DT_INST_PROP(0, size),
```

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]

File: drivers/eeprom/eeprom_mchp_xec.c (hunk 1 above)
Comment (Manimaran-A) [reply drivers/eeprom/eeprom_mchp_xec.c]: Updated with "DT_INST_PROP"

**Review by henrikbrixandersen — APPROVED** [review]

**Review by jvasanth1 — APPROVED** [review]

**Comment by fabiobaltieri** [comment]
Hey just wanted to point out, commit message first line should describe the change, "Bug fix" is way too generic, the majority of the commit are fixing bugs. :-) See https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-message-guidelines.

Merging anyway, just something to keep in mind.

**Comment by Manimaran-A** [comment]
Ok, i ensure on the next commit

**Pull request merged**